include/oc_helpers.h

```diff
@@ -36,6 +37,13 @@
 extern "C" {
 #endif
 
+typedef struct oc_string64_s
+{
+  struct oc_string64_s *next;
```

Comment:
Do you have some future use in mind where `oc_string64_t` is put into an `oc_list_t`? From the uses in this PR we don't need that, so the `next` member is not necessary, in fact we skip the whole type and just use `char[64]`.

Reply:
removed next + moved to internal header.